cmd/internal/cli/pull.go

```diff
@@ -6,14 +6,24 @@
 package cli
 
 import (
+	"context"
 	"fmt"
 	"io"
 	"os"
 	"os/signal"
 	"path/filepath"
 	"syscall"
 
+	"strings"
```

Comment:
Small comment, this should be one line up (no whitespace btw `"syscall"` and `"strings"`)